keps/sig-node/1342-kubelet-container-resources-cri-api-changes/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Container Resources CRI API Changes for Pod Vertical Scaling
+kep-number: 1342
```

Comment:
This shouldn't be 1342 (same for directory).
The number (being also prefix of directory) should be the number of corresponding issue in the repo (and I think there isn't yet any for this, so a new one should be opened).

Reply:
Thanks @wojtek-t for the review. I found that this file is added in the PR #1342 . Is there any way to find the issue for corresponding kep ? I checking the format for opening the issue for this.

Reply:
I was trying to look for it too and I didn't find any. So I'm almost sure there simply isn't any and you probably need to open a dedicated one.

Reply:
Thanks for the info. I have created an issue for this: https://github.com/kubernetes/enhancements/issues/2273

Reply:
Can you please review the issue  #2273 and this PR.

Reply:
This looks fine to me - though you need approval from SIG node leads anyway.